### DISCUSSION TITLE
Performance panel bug with ToggleBool

### DISCUSSION THREAD

**Discussion by bejucoo** [cables-gl/cables_docs#455] [! Welcome - readme first!]
Hello,

when I use ToogleBool with KeyPressLearn to show/hide the performance panel using the 'P' key, it shows up incorrectly:

![Screenshot_20230109_114836](https://user-images.githubusercontent.com/19770567/211362010-ee01b46c-85d9-4990-a12b-fa3207fb466d.png)

When I click it to open it and hide it pressing the key, then it works fine:

![Screenshot_20230109_115056](https://user-images.githubusercontent.com/19770567/211362381-2771d6db-0f63-49a6-b6dd-6bfbc842494f.png)

**Comment by pandrr** [comment]
i tested this quickly, worked for me, please create example

**Reply by bejucoo** [reply]
Hmm, it happens only in the editor. If i export the patch or watch it live it works properly. I made this example: https://cables.gl/p/9AATWm

It may be my machine as well.

**Reply by bejucoo** [reply]
Yes, only in editor. Tested in Firefox and Chrome. The steps I follow are:
- Click toggle button
- Click performance bar to 'open' it
- Click performance bar 'close' it
- Click toggle button
- Click toggle button
